content/when-should-we-automate-tests.md

```diff
@@ -0,0 +1,34 @@
+Title: When should we automate tests
+Date: 2022-01-01
+Category: Posts
+Tags: learning, testing, devops
+Slug: when-should-we-automate-tests
+Author: Aditya Chourasiya
+
+Summary: This article is gist of brainstorming session among QA team(Mohita Kataria, Midhi Naidu and Aditya Chourasiya) at WorkSafeBC
+
+   Automation is advised when tests are **prone to human error** or **complex** or if they need **repeating**, **reporting** , **execution speed** and ofcourse fetch good **return on investment**. 
+   
+   On the conterary, test cases for aesthetics, accessibility, one-time, urgent non-regression(hotfixes), exploratory, with unpredictable results and/or with less return on investment,  should not be automated. 
+   
+<img src="../images/chapline-automation.gif">
```

Comment:
@adichourasiya  can you change this line to:  ![Chaplain](../images/chapline-automation.gif)